### DISCUSSION TITLE
When will next release be?

### DISCUSSION THREAD

**Discussion by RasmusBergHogia** [6pac/SlickGrid#631] [Q&A] [answered]
I wonder when next release will be? 
My company and I wait on the security vulnerability fix for CSP to be release.

Best regards,
Rasmus Berg

**Comment by ghiscoding** [comment, marked as answer]
Probably by end of the week or next week but you'll probably need a new version of Aurelia-Slickgrid also and I'm not planning this until at least next week if not more

**Reply by ghiscoding** [reply]
@RasmusBergHogia FYI, I also released Aurelia-Slickgrid `3.10.1` tonight which include your commits and a lot more

**Reply by RasmusBergHogia** [reply]
Thanks for the update

**Comment by NadavPopplewell** [comment]
Is there a reason why the Latest release (as shown in the main repository page) is 2.4.3?

**Reply by ghiscoding** [reply]
that's not exactly true, `2.4.3` is the last version that 6pac specifically wrote something in the version (he doesn't anymore), however if you click on the "Releases (85)" then you'll see all releases including the latest 
![image](https://user-images.githubusercontent.com/643976/134352990-cc9b5880-5e1c-440f-b21f-544888245871.png)

**Comment by NadavPopplewell** [comment]
v2.4.3 is marked as the Latest version, and it shows as the latest version in the repository main page.
Are the other releases stable?
If so, then why is the LATEST release an old one (from 37 releases ago)?

**Comment by 6pac** [comment]
Thanks for bring that to our attention. It [appears](https://github.community/t/why-last-release-is-not-marked-as-latest-release/421) that if I am releasing locally and pushing to GitHub (rather than creating the release on GitHub) that it doesn't automatically update the 'Latest' tag. I'll try adding a Title to the release script and see if that works next time around.

**Reply by 6pac** [reply]
... hmm it's a bit complicated though. Git supports tagging but not releases, that's a GitHub-only thing, so I'm going to have to connect to the GitHub API to tweak the release. It looks like GitHub creates a release automatically from the tag, but doesn't configure it correctly for it to be recognised fully as a release.
